samples/model/README.md

```diff
@@ -0,0 +1,49 @@
+# Model Sample
+
+This sample demonstrates how to start models/create a model using your own model script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](To do).
+
+## Preparing the Model Script
```

Comment:
let's remove this line, it is confusing. content of this section fit fine under prereqs